starfish/pipeline/features/pixels/_base.py

```diff
@@ -0,0 +1,7 @@
+from starfish.pipeline.algorithmbase import AlgorithmBase
+
+
+class PixelFinderAlgorithmBase(AlgorithmBase):
+    def find(self, stack):  # -> Tuple[SpotAttributes, EncodedSpots]:
```

Comment:
Does this not work as a return type?

Reply:
Nope, needs #226, this does not return EncodedSpots or true SpotAttributes (although it returns something _like_ SpotAttributes). I've removed the comment.